core/security.js

```diff
@@ -0,0 +1,37 @@
+class Security {
+  constructor() {
+    this.supportsTrustedTypes =
+      window.trustedTypes && window.trustedTypes.createPolicy;
+    let trustedTypesPolicy;
+
+    try {
+      if (this.supportsTrustedTypes) {
+        trustedTypesPolicy = window.trustedTypes.createPolicy('quill@core', {
+          createHTML: function (html) {
+            // Trust all strings passed to this function.
+            return html;
+          },
+        });
+      }
+    } catch (e) {}
+
+    this.blessHTML = trustedTypesPolicy ? trustedTypesPolicy.createHTML.bind(trustedTypesPolicy) : (x) => x;
```

Comment:
This is a matter of taste, but I prefer the following pattern:
```
let trustedTypesPolicy = {
  createHTML: function (html) {
    // Trust all strings passed to this function.
    return html;
  } 
};

try {
  if (this.supportsTrustedTypes) {
    trustedTypesPolicy = window.trustedTypes.createPolicy('quill', trustedTypesPolicy);
  }
} catch (e) {}

this.blessHTML = trustedTypesPolicy.createHTML.bind(trustedTypesPolicy);
```

Slightly more compact and also gets rid of `(x) => x`, which may not be as portable depending on if Quill uses a JS bundler.